recipes/geotrans/all/conandata.yml

```diff
@@ -1,7 +1,12 @@
 sources:
+  "3.9":
+    url: "https://earth-info.nga.mil/php/download.php?file=wgs-mastertgz"
```

Comment:
@uilianries one of the problems is that this download link isn't versioned. This means that if there's a new release, geotrans 3.9 will fail. The previous solution to this was to mirror the source archive on the c3i artifactory server (see the 3.8 source url). Other solutions present challenges because this is approx 153MB.

Are you able to mirror these sources to the c3i artifactory to avoid this problem?

Reply:
Done.